core/src/main/scala/org/broadinstitute/dsp/workbench/welder/Crc32c.scala

```diff
@@ -38,7 +38,7 @@ object Crc32c {
   private val BYTE_MASK = 0xff
 
   def calculateCrc32ForFile(path: java.nio.file.Path, blocker: Blocker)(implicit cs: ContextShift[IO]): IO[Crc32] =
-    fs2.io.file.readAll[cats.effect.IO](path, blocker, 4096).compile.to[Array].map(bytes => calculateCrc32c(bytes))
+    fs2.io.file.readAll[cats.effect.IO](path, blocker, 4096).compile.to(Array).map(bytes => calculateCrc32c(bytes))
```

Comment:
Is this an `fs2` change? I'm used to `to[A]`

Reply:
yea...I think it's a new scala thing